lib/rubycritic/command_factory.rb

```diff
@@ -8,16 +8,16 @@ def self.create(options = {})
       case Config.mode
       when :version
         require "rubycritic/commands/version"
-        Command::Version.new
+        Command::Version.new(options_hash)
```

Comment:
Let's not use [hungarian notation](https://en.wikipedia.org/wiki/Hungarian_notation) - I think just calling it "options" is fine here.